### PR TITLE
Regenerate spanner GAPIC

### PR DESCRIPTION
**Pull request by geigerj**
Includes some hand-edits to codegen to make new GAPIC credentials work. We'll make corresponding updates to the generator.

### CODE REVIEW

File: google-cloud-spanner/lib/google/cloud/spanner/admin/database/v1/doc/google/iam/v1/policy.rb [hunk 1]
```diff
@@ -121,19 +121,7 @@ class PolicyDelta; end
       #     A single identity requesting access for a Cloud Platform resource.
       #     Follows the same format of Binding.members.
       #     Required
-      class BindingDelta
-        # The type of action performed on a Binding in a policy.
-        module Action
```
Comment (geigerj) [inline google-cloud-spanner/lib/google/cloud/spanner/admin/database/v1/doc/google/iam/v1/policy.rb]: @eoogbe Could you check why we're no longer generating this enum type and file an issue in `toolkit` if this is unexpected?

File: google-cloud-spanner/lib/google/cloud/spanner/admin/database/v1/doc/google/spanner/admin/database/v1/spanner_database_admin.rb [hunk 2]
```diff
@@ -55,22 +66,22 @@ module State
           # @!attribute [rw] page_token
           #   @return [String]
           #     If non-empty, +page_token+ should contain a
-          #     Next_page_token from a
-          #     previous ListDatabasesResponse.
+          #     {Google::Spanner::Admin::Database::V1::ListDatabasesResponse#next_page_token Next_page_token} from a
```
Comment (blowmage) [inline google-cloud-spanner/lib/google/cloud/spanner/admin/database/v1/doc/google/spanner/admin/database/v1/spanner_database_admin.rb]: The link description has a case mistake. The method is `next_page_token`, not `Next_page_token`.

Comment (geigerj) [reply google-cloud-spanner/lib/google/cloud/spanner/admin/database/v1/doc/google/spanner/admin/database/v1/spanner_database_admin.rb]: Acknowledged. Filed https://github.com/googleapis/toolkit/issues/1568 to fix in the generator.

**Comment by geigerj** [comment]
Superseded by #1804.

**Pull request closed without merging**